sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/base/BaseFragment.kt

```diff
@@ -0,0 +1,90 @@
+package co.nimblehq.sample.compose.ui.base
```

Comment:
As this class belongs to `XML`, we should remove this class, right? 🙏

Same goes for `BaseFragmentCallbacks.kt`